projects/ion/src/lib/chip/chip.component.spec.ts

```diff
@@ -54,6 +60,15 @@ describe('ChipComponent', () => {
     }
   );
 
+  it.each(['left'])(
+    'should render icon on left',
+    async (iconPosition: Direction = 'left') => {
+      await sut({ label: 'custom-position', iconPosition });
+      const element = screen.getByText('custom-position');
+      expect(element).toHaveClass('positionIcon');
```

Comment:
We use `it.each` when we have more than one cases to test. I don't think you need to use it here.
```suggestion
  it(
    'should render icon on left',
    async () => {
      await sut({ label: 'custom-position', iconPosition: 'left' });
      const element = screen.getByText('custom-position');
      expect(element).toHaveClass('positionIcon');
```